### PR TITLE
doc: added documentation for the Color Chooser

### PR DESCRIPTION
**Pull request by LarsGit223**
I added a little section documenting the Colour Chooser to the manual. Also see #2204.

### CODE REVIEW

File: doc/geany.txt [hunk 1]
```diff
@@ -1145,6 +1145,17 @@ applications, in that you'll need to continue to hold down the Ctrl and Shift
 keys while typing the code point hex digits (and the Enter or Return to finish the code point).
 
 
+Inserting color values
+^^^^^^^^^^^^^^^^^^^^^^
+
+You can insert a color value by selecting *Tools->Colour Chooser* from the menu.
```
[the remark this answers is not in the document]
Comment (LarsGit223) [reply doc/geany.txt]: But the menu item not. This is taken from the menu as-is.

File: doc/geany.txt [hunk 2]
```diff
@@ -1145,6 +1145,17 @@ applications, in that you'll need to continue to hold down the Ctrl and Shift
...
+A dialog appears to select the wanted color. On clicking on *Apply* or *Select*
+the code for the choosen color will be inserted in the format *#RRGGBB*.
```
[the remark this answers is not in the document]
Comment (LarsGit223) [reply doc/geany.txt]: Thanks, fixed.

File: doc/geany.txt (hunk 1 above)
[the remark this answers is not in the document]
Comment (LarsGit223) [reply doc/geany.txt]: Forget it. Just noticed my desktop language was indeed set to ```English (United Kingdom)```.

[the remark this answers is not in the document]
Comment (LarsGit223) [reply doc/geany.txt]: Fixed now.

**Comment by LarsGit223** [comment]
> Could also mention that if the cursor is next to/within a hash-prefixed hex code, then the dialog will display the color the code represents.

Done.

**Comment by ntrel** [comment]
Great, thanks!

BTW you can add extra commits with follow up changes, then we'll squash them when we merge the pull. Rather than force pushing with a small change. Then an existing review stands and any new reviewer knows only the new commit(s) need particular scrutiny.

**Pull request merged**